rust/src/tests/truncate.rs

```diff
@@ -79,3 +81,7 @@ fn unchanged_ctime_failed(ctx: &mut SerializedTestContext) {
         });
     });
 }
+
+// (f)truncate/04.t
```

Comment:
Should there be other tests in ftruncate.rs?

Reply:
The original suite tested only for truncate, even in ftruncate files. Should I add ftruncate too?

Reply:
What about files like tests/ftruncate/00.t ?  Doesn't that test ftruncate?

Reply:
Yes, it seems to be the only one with the 13.t. I'm not sure if it's supposed to be like this, or just a copy/paste error.

Reply:
Oh, I see now.  yeah, that definitely looks like a copy/paste bug in the original.

Reply:
Alright, I will add the `ftruncate` tests then!

Reply:
I think that I understand now why there wasn't tests for ftruncate. I don't think that it's possible to create files if the path is invalid...